api/src/pcapi/sandboxes/scripts/getters/pro.py

```diff
@@ -44,10 +46,13 @@ def create_regular_pro_user_with_virtual_offer() -> dict:
 
 
 def create_adage_environnement() -> dict:
-    current_year = educational_factories.EducationalCurrentYearFactory()
-    next_year = educational_factories.EducationalYearFactory()
-
-    offer = educational_factories.CollectiveOfferTemplateFactory(name="Mon offre collective")
+    current_year = educational_factories.EducationalCurrentYearFactory(adageId=random.randint(1, 100000000))
```

Comment:
Les tests adage ont échoué:
`une erreur s'est produite lors du calcul de \"pro create_adage_environnement\" pour la sandbox : (psycopg2.errors.UniqueViolation) duplicate key value violates unique constraint \"ix_educational_institution_institutionId\"\nDETAIL:  Key (\"institutionId\")=(0910620E) already exists.\n\n[SQL: INSERT INTO educational_institution (\"institutionId\", \"institutionType\", name, city, \"postalCode\", email, \"phoneNumber\", \"isActive\", \"ruralLevel\") VALUES (%(institutionId)s, %(institutionType)s, %(name)s, %(city)s, %(postalCode)s, %(email)s, %(phoneNumber)s, %(isActive)s, %(ruralLevel)s) RETURNING educational_institution.id]\n[parameters: {'institutionId': '0910620E', 'institutionType': 'COLLEGE', 'name': 'DE LA TOUR2', 'city': 'PARIS', 'postalCode': '75000', 'email': 'de-la-tour2@paris.fr', 'phoneNumber': '0600000000', 'isActive': True, 'ruralLevel': None}]\n(Background on this error at: https://sqlalche.me/e/14/gkpj)"`